### PR TITLE
Redesign job, status

### PR DESCRIPTION
**Pull request by swashcap**
_WIP!_

image

### CODE REVIEW

**Comment by cdaringe** [comment]
: 🔥

File: src/configure.ts [hunk 1]
```diff
@@ -26,6 +28,20 @@ export const upsertConfigDir = async () => {
 export const edit = (electron: typeof Electron) =>
   electron.shell.showItemInFolder(getConfigFilename())
 
+export const openLogFile = async (electron: typeof Electron) => {
+  const files = await promisify<string, string[]>(readdir)(getConfigDir())
```
Comment (cdaringe) [inline src/configure.ts]: fs.promises.readdir is a think if you don't want to p'fiy

File: src/bin.ts [hunk 2]
```diff
@@ -45,8 +47,12 @@ app.on('ready', async () => {
     switch (msg) {
       case FromUi.REQUEST_OPEN_CONFIG_FOLDER:
         return edit(electron)
+      case FromUi.REQUEST_OPEN_LOG_FILE:
+        return openLogFile(electron)
       case FromUi.LOG:
         return processLog(payload)
+      case FromUi.REQUEST_OPEN_ISSUE_URL:
+        return shell.openExternal(pkg.bugs)
```
Comment (cdaringe) [inline src/bin.ts]: cool. v cool even

**Comment by cdaringe** [comment]
even if you're not done... i want it!

**Comment by swashcap** [comment]
_shit, dawg_ ☺️ 

Will rebase and get these additional things in ( 👻_dark mode_🎃)

**Pull request merged**